eva-jooq/src/main/kotlin/com/razz/jooq/dsl/DSLs.kt

```diff
@@ -16,8 +16,8 @@ object ArrayDSL {
 
 object JsonDSL {
 
-    fun jsonbContainsKey(field: Field<JSONB>, key: String) =
-        DSL.field("{0} ? {1}", Boolean::class.java, field, key)
+    fun jsonbContainsKeys(field: Field<JSONB>, vararg keys: String) =
+        DSL.condition("jsonb_exists_all({0}, {1}::text[])", field, array(keys))
```

Comment:
Should we delete old implementation?

Reply:
yes, it doesn't work and nobody could use it

https://stackoverflow.com/questions/38370972/how-do-i-use-postgresql-jsonb-operators-containing-a-question-mark-via-jdb/38370973#38370973